src/templates/index.html

```diff
@@ -53,13 +88,40 @@
 
     <div id="sidebar" class="leaflet-sidebar collapsed">
         <!-- Nav tabs -->
+        
         <div class="leaflet-sidebar-tabs">
             <ul role="tablist"> <!-- top aligned tabs -->
-                <li><a href="#home" role="tab"><i class="fa fa-info-circle"></i></a></li>
+                <li><a href="#uploadFile" role="tab"><i class="fa fa-upload"></i></a></li>
+                <li><a href="#home" role="tab"><i class="fa fa-info-circle"></i></a></li>           
             </ul>
         </div>
+
 <!-- Tab panes -->
+
 <div class="leaflet-sidebar-content">
+    <div class="leaflet-sidebar-pane" id="uploadFile">
+        <h1 class="leaflet-sidebar-header">
+            <div class="leaflet-sidebar-close"><i class="fa fa-caret-left"></i></div>
+        </h1>
+        <div class="leaflet-sidebar-content" style="padding-left: 20px;">
+            <h2>Input Excel File </h2>
+            <!--p>Please upload an Excel file adding relevant details 
+                that include Latitude and Longitude of the location, Hose Length(m),
+                Transport mode and Point Type.  </p-->
+            <p>The application requires the uploaded excel file, adhering to a specific data structure. 
+                Therefore, we recommend downloading our template <a href="https://drive.google.com/uc?export=download&id=12pwJ57A27z_MDwMeAm_Hl4W4nlI6a4Bz" download>(Click here)</a>
+                which includes all necessary inputs and data validation rules.
```

Comment:
In testing this functionality, I filled in a template (link below) and when i uploaded it, I got the error message telling me it as not a valid excel file. Can you troubleshoot this example file?
[FileTemplate (1).xlsx](https://github.com/DSSGxMunich/hydroxplorer-web-app/files/13917709/FileTemplate.1.xlsx)

Reply:
![image](https://github.com/DSSGxMunich/hydroxplorer-web-app/assets/60874029/5636943d-d9b1-4ceb-ba6f-afc0722d15fa)
I had uploaded the same file from my end, and it seems working.
Please find the attached image for reference.